packages/core/components/DraggableComponent/index.tsx

```diff
@@ -97,6 +97,7 @@ export const DraggableComponent = ({
               {label && (
                 <div className={getClassName("actionsLabel")}>{label}</div>
               )}
+              <div id={`DraggableComponentToolbarPortal-${id}`} />
```

Comment:
What's the purpose of this portal? To inject new actions into the toolbar?

If so, I think a first party API might be a cleaner way of doing this instead of asking the user to inject via a portal. Something like `<Puck toolbarActions={<div />} />`

Reply:
I want to be able to inject from the inline editor. eg https://inline-rich-text.vercel.app/edit (click on the heading).
In the above example they are at the end of toolbar, but I think the middle is better (that was just the easiest for me to target with getElement / css Query hack).
These are dynamically changing based on lexical plugins.
An alternative would be to set them via setUI, but I do think the portal option is much cleaner for my particular use case.

Reply:
I think this API needs a bit more thought before we move forwards:

1. I don't think we should be using portals for some elements of custom UI, and props for others.
2. Users may wish to remove existing overlay controls (possibly a separate, but related API)

I'm not tracking this here: https://github.com/measuredco/puck/issues/377

Reply:
I would still like the (amended) UI fix for the dropdown, which I can put as a separate PR.

#377 is good, but... I would still at the least like a stable data-* attribute I can target in the mean time, even if it's an experimental, undocumented solution.